app/views/planning_applications/validation/development_type_reportings/_form.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= form_with model: @planning_application, local: true, url: planning_application_validation_development_type_reporting_url(@planning_application) do |form| %>
+  <%= form.govuk_error_summary(presenter: ErrorPresenter) %>
+
+  <%= form.govuk_radio_buttons_fieldset(
+      :development_type_for_reporting,
+      legend: { size: "m" }
+    ) do %>
+      <% @planning_application.fetch_development_types_for_reporting.each do |type| %>
```

Comment:
If we structure the data correctly in the locale file you can simplify this like so:

``` erb
<% t(@planning_application.application_type_name, scope: :development_types).each do | type, category, label %>
  <%= form.govuk_radio_button(:development_type, category, label: { text: label }, data: { "aria-controls": "conditional-#{type}" }) %>
<% end %>
```

``` yaml
en:
  development_types:
    - ["lawfulness_certificate", "Q26", "Certificate of Lawful Development"]
```

I probably haven't got the exact structure right, but you should get the idea.

Reply:
I also don't like this type of format and tried to think a better one but unfortunately there are more fields like `guidance`, `guidance_link` and `legislation`